extensions/resteasy-reactive/rest-jackson/deployment/src/main/java/io/quarkus/resteasy/reactive/jackson/deployment/processor/JacksonCodeGenerator.java

```diff
@@ -304,6 +304,9 @@ private String fieldName() {
 
         private String fieldNameFromMethod(MethodInfo methodInfo) {
             String methodName = methodInfo.name();
+            if (methodName.length() <= 3) {
```

Comment:
```suggestion
            if (methodName.equals("get") || methodName.equals("set") || methodName.equals("is")) {
```

Reply:
Can I ask you why this solution is better? Is it for better readability?
Thanks for the review.